src/cmd-build

```diff
@@ -118,24 +109,26 @@ ostreesetup --nogpg --osname=coreos --remote=coreos --url=file:///mnt/ostree-rep
 EOF
 
 imageprefix=${name}-${version}-${image_genver}
-/usr/lib/coreos-assembler/virt-install --dest=$(pwd)/${imageprefix}-base.qcow2 \
-               --create-disk --kickstart $(pwd)/local.ks --kickstart-out $(pwd)/flattened.ks \
+/usr/lib/coreos-assembler/virt-install --dest=$(pwd)/tmp/${imageprefix}-base.qcow2 \
+               --create-disk --kickstart $(pwd)/tmp/local.ks --kickstart-out $(pwd)/tmp/flattened.ks \
                --location ${workdir}/installer/*.iso --console-log-file $(pwd)/install.log \
                --local-repo=${workdir}/repo
 
-/usr/lib/coreos-assembler/gf-oemid ${imageprefix}-base.qcow2 $(pwd)/${imageprefix}-qemu.qcow2 qemu
+/usr/lib/coreos-assembler/gf-oemid tmp/${imageprefix}-base.qcow2 $(pwd)/${imageprefix}-qemu.qcow2 qemu
```

Comment:
should the second arg be put in tmp or not? `$(pwd)/${imageprefix}-qemu.qcow2` ?

Reply:
I just didn't know if it was a mistake or not. If it was intentional then +1

Reply:
yeah I see now. I didn't realize  that everything we put in the current directory (which is `${tmp_builddir}`) is everything we want to keep